src/components/NavBar.vue

```diff
@@ -35,7 +35,7 @@
             <span>Tools</span>
           </div>
         </div>
-        <div class="navbar-buttonWrapper desktop" @click="editProfile" data-cyId="editProfile">
+        <div class="navbar-buttonWrapper desktop" @click="openProfile" data-cyId="openProfile">
```

Comment:
`data-cyId` used here and in other places in this file refers to a "cypress ID," which is an attribute that we use to retrieve this element when running our front-end cypress tests. 

I believe the tests are failing because you changed the name here. If you change it back, they should pass, as the new tools page is behind a gatekeep, so your code should not cause a test to fail (until the gatekeep is removed, at which point we can talk about rewriting it 😄)